File: beer.jl
using Twitter

#認証
twitterauth(
    "XXXXXXXXXXXXXXXXX", # Consumer key
    "XXXXXXXXXXXXXXXXX", # Consumer secret
    "XXXXXX-XXXXXXXXXX", # Access token
    "XXXXXXXXXXXXXXXXX") # Access token secret

# 連続投稿
for num in 1:rand(1:10)
    println(num)
    # ツイート本文
    sentence = "そんなことより" * join(["🍺" for i in 1:rand(1:10)]) * ["", "飲むもじゃ", "飲むもじゃよ"][rand(1:3)]
    # ツイート投稿
    post_status_update(sentence)
    sleep(1)
end

# 参考
# https://github.com/randyzwitch/Twitter.jl

File: beer.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

import time
import random
from requests_oauthlib import OAuth1Session

#認証
CK = 'XXXXXXXXXXXXXXXXX' # Consumer Key
CS = 'XXXXXXXXXXXXXXXXX' # Consumer Secret
AT = 'XXXXX-XXXXXXXXXXX' # Access Token
AS = 'XXXXXXXXXXXXXXXXX' # Accesss Token Secert

# ツイート投稿用のURL
url = "https://api.twitter.com/1.1/statuses/update.json"

# 連続投稿
for num in range(random.randrange(1,10)):
	print(num)
	# ツイート本文
	sentence = 'そんなことより' + random.randrange(1,10) * '🍺' + random.choice(['', '飲むもじゃ', '飲むもじゃよ'])
	params = {"status": sentence}

	# OAuth認証で POST method で投稿
	twitter = OAuth1Session(CK, CS, AT, AS)
	# ツイート投稿
	req = twitter.post(url, params = params)

	# レスポンスを確認
	if req.status_code == 200:
	    print ("OK")
	else:
	    print ("Error: %d" % req.status_code)
	time.sleep(1)

# 参考
# https://qiita.com/yubais/items/dd143fe608ccad8e9f85